### PR TITLE
issue #3317 resolved. Update required documentation

### PR DESCRIPTION
**Pull request by OzairKhanBro**
Now we will be able to add resources and file by using '.' as well as '/'.

### CODE REVIEW

**Comment by pekkaklarck** [comment]
This PR seems to lack tests altogether. They need to be added before it can be merged or even more thoroughly reviewed.

Unfortunately this is too late for RF 3.2.

**Comment by OzairKhanBro** [comment]
Can you please tell me where I should add test cases. Either in [test_importer_util.py](https://github.com/robotframework/robotframework/blob/75432ef6005caf9b133b5c8565152d3dd1516a46/utest/utils/test_importer_util.py#L112) or in [test_imports.py](https://github.com/robotframework/robotframework/blob/75432ef6005caf9b133b5c8565152d3dd1516a46/utest/running/test_imports.py#L46)

**Comment by pekkaklarck** [comment]
Most importantly some acceptance tests are needed.

File: doc/userguide/src/CreatingTestData/ResourceAndVariableFiles.rst [hunk 1]
```diff
@@ -33,8 +33,9 @@ where the importing file is located. If the file is not found there,
 it is then searched from the directories in Python's `module search path`_.
 The path can contain variables, and it is recommended to use them to make paths
 system-independent (for example, :file:`${RESOURCES}/login_resources.robot` or
-:file:`${RESOURCE_PATH}`). Additionally, forward slashes (`/`) in the path
-are automatically changed to backslashes (:codesc:`\\`) on Windows.
+:file:`${RESOURCE_PATH}`). Additionally, forward slashes (`/` or `.`) in the path
+are automatically changed to backslashes (:codesc:`\\`) on Windows. In case of
+(`.`) last . is not going to replac.
```
Comment (miktuy) [inline doc/userguide/src/CreatingTestData/ResourceAndVariableFiles.rst]: mb you mean to replac**e**?

**Comment by pekkaklarck** [comment]
I close this PR because there has been no activity since 2019. The original issue #3317 is valid so I leave it open. Notice that implementing it probably wouldn't require changes in the code that was changed in this PR.

**Pull request closed without merging**